hoodie-client/src/main/java/com/uber/hoodie/config/HoodieWriteConfig.java

```diff
@@ -70,7 +70,8 @@
   private static final String DEFAULT_CONSISTENCY_CHECK_ENABLED = "false";
   private static final String EMBEDDED_TIMELINE_SERVER_ENABLED = "hoodie.embed.timeline.server";
   private static final String DEFAULT_EMBEDDED_TIMELINE_SERVER_ENABLED = "false";
-
+  private static final String FAIL_ON_SINGLE_ARCHIVING_ENABLED_PROP = "hoodie.fail.on.single.archiving";
```

Comment:
rename everything (variable, property, default) to FAIL_ON_TIMELINE_ARCHIVING?

Reply:
Thanks! 
Renamed everything...

I added short description of what we do here: https://github.com/apache/incubator-hudi/issues/661#issuecomment-504168231

:)

Reply:
and we are using hudi 0.4.7 with Spark 2.4.3